projects/Reth_contributor.md

```diff
@@ -0,0 +1,104 @@
+# Project Proposal
+
+Become a core contributor for [Reth](https://github.com/paradigmxyz/reth), a new Ethereum execution client written in Rust!
+
+## Motivation
+
+Client diversity is one of the most important features of Ethereum as it helps to further decentralise the network. Right now there are already several implementations for the Ethereum protocol, divided in consensus and execution clients, as you can see from the image below.
+
+![client_diversity.png](Reth_contributor/client_diversity.png)
+
+Reth is an execution client and it’s the first one written in the Rust language, another good thing since it creates diversity also on the programming languages used for Ethereum clients. It aims to be very fast (Rust language is known for its speed and reliability) and optimised in order to be able to satisfy different users:
+
+- big companies who need power and speed, have lots of money to invest in machines and infrastructures.
+- normal users who just want to run their own node with a normal computer.
+
+I am really excited to contribute to such a significant software project. Working on it lets me study and gain deep knowledge of various aspects of the Ethereum protocol.
+
+## Project description
+
+My project for this EPF is divided into two parts:
+
+1. Create an implementation of [EIP-7212](https://eips.ethereum.org/EIPS/eip-7212) for Reth. This EIP aims to introduce a precompile for elliptic curve operations on the ”*sepc256r1*” curve, which is different from the ”*sepc256k1*” used on Ethereum (and Bitcoin) for signatures verification.
+2. Solve a lot of smaller issues on the Reth repo, covering various aspects of the protocol and of an Ethereum node.
+
+## Specification
+
+### EIP-7212
```

Comment:
Is there an issue in reth repo tracking 7212 implementation? Might be worth linking here

Reply:
There is no issue right now on Reth for it but I can create it. I'll update it with the issue link when it will be online